server/moodle/mod/livequiz/index.php

```diff
@@ -36,3 +36,6 @@
 require_course_login($course);
 
 $modinfo = get_fast_modinfo($course);
+
+
+// Make funtion for displaying activity information here (empty function removed cus it caused errors).
```

Comment:
This used to be an empty for loop with the first part of the comment inside it, 
But since code sniffer doesn't allow empty for loop we removed it, I do not know what activity information this refers to so we left it here in case we need to go back to it